resources/linux/code.appdata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
```

Comment:
I realize I was ambiguous, I actually meant make another PR with appdata.xml and I'll merge that instantly :stuck_out_tongue: